google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/BigtableTableAdminClient.java

```diff
@@ -177,13 +178,24 @@ public void close() {
    * <p>Sample code:
    *
    * <pre>{@code
+   * // A table with a single column family, which retains only the latest value.
    * Table table = client.createTable(
    *   CreateTableRequest.of("my-table")
-   *     .addFamily("cf2", GCRULES.maxVersions(1)
+   *     .addFamily("cf2", GCRULES.maxVersions(1))
+   * );
+   *
+   * // Another table with more complex garbage collection rules.
+   * Table table = client.createTable(
+   *   CreateTableRequest.of("my-table")
+   *     .addFamily("cf2", GCRULES.union()
+   *       .rule(GCRULES.maxAge(Duration.ofDays(30)))
+   *       .rule(GCRULES.maxVersions(5))
+   *     )
```

Comment:
extra closing parent

Reply:
I added this parenthesis for addFamily(as this has a multi-line argument).

Reply:
my mistake